File: Python/1072 - Intervalo 2.py
n = int(input())
valores = [int(input()) for _ in range(n)]

in_intervalo = sum(1 for v in valores if 10 <= v <= 20)
out_intervalo = n - in_intervalo

print(f"{in_intervalo} in")
print(f"{out_intervalo} out")

File: JavaScript/1072 - Intervalo 2.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8').trim().split('\n');
var N = parseInt(input[0]);
var valores = input.slice(1).map(Number);

var dentro = valores.filter(v => v >= 10 && v <= 20).length;
var fora = N - dentro;

console.log(`${dentro} in`);
console.log(`${fora} out`);